src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```diff
@@ -332,6 +332,18 @@ ul.summary-list > li {
     margin-top:0;
     margin-bottom:1px;
 }
+ul.see-list, ul.see-list-long {
+    padding-left: 0;
+    list-style: none;
+}
+ul.see-list li {
+    display: inline;
+}
+ul.see-list li:not(:last-child):after,
+ul.see-list-long li:not(:last-child):after {
+    content: ", ";
+    white-space: pre-wrap;
+}
```

Comment:
Cool! I didn't know you could do that. Is this widely supported?

Reply:
According to MDN documentation all used pseudo-classes/elements are supported in all browsers including Internet Explorer 9.

https://developer.mozilla.org/en-US/docs/Web/CSS/:not
https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child
https://developer.mozilla.org/en-US/docs/Web/CSS/::after

 I have tested the pages on my old laptop using IE 11 and Edge 44. These are the oldest browsers I have access to, they both rendered both list styles correctly.